src/sentry/tasks/scheduler.py

```diff
@@ -0,0 +1,40 @@
+
+from __future__ import absolute_import, division
+
+import logging
+
+from django.utils import timezone
+
+from sentry.models import ScheduledJob
+from sentry.tasks.base import instrumented_task
+from sentry.utils.locking import UnableToAcquireLock
+from sentry.utils import redis
+from sentry.utils.locking.backends.redis import RedisLockBackend
+from sentry.utils.locking.manager import LockManager
+
+logger = logging.getLogger('sentry')
+
+
+@instrumented_task(name='sentry.tasks.enqueue_scheduled_jobs')
+def enqueue_scheduled_jobs(**kwargs):
+    from sentry.celery import app
+
+    lock_key = 'scheduler:process'
+    try:
+        locks = LockManager(RedisLockBackend(redis.clusters.get('default')))
```

Comment:
What @mattrobenolt said.

If you want to maintain the `nowait=True` behavior from the previous change, you don't have to wrap the `lock.acquire` in the retry policy and can instead do this:

```python
with locks.get('scheduler:process', duration=60).acquire():
```